### PR TITLE
Edit and expand command docs

### PR DESCRIPTION
**Pull request by jbogarthyde**
This one is just the top-level commands. Will do separate PR for subcommands (generation schematics).

### CODE REVIEW

File: packages/angular/cli/commands/run.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/schema",
   "$id": "ng-cli://commands/run.json",
-  "description": "Runs a custom target defined in your project.",
+  "description": "Runs an architect target with a custom build configuration defined in your project.",
```
Comment (alan-agius4) [inline packages/angular/cli/commands/run.json]: Note sure about the terminology here, if it should be `build` or `builder`

```
custom build configuration -> custom builder configuration
```

File: packages/angular/cli/commands/run.json [hunk 2]
```diff
@@ -15,14 +15,14 @@
...
-          "description": "The target to run.",
+          "description": "The command to run.",
           "$default": {
             "$source": "argv",
             "index": 0
           }
         },
         "configuration": {
-          "description": "Specify the configuration to use.",
+          "description": "A named build configuration, defined in the \"configurations\" section of angular.json.\nThe build uses the configuration associated with the given target.",
```
Comment (alan-agius4) [inline packages/angular/cli/commands/run.json]: same
```
A named build configuration - > A named builder configuration
The build uses -> The builder uses
```

File: packages/angular/cli/commands/definitions.json [hunk 3]
```diff
@@ -14,14 +14,14 @@
...
-          "description": "A named configuration environment, as specified in the \"configurations\" section of angular.json.",
+          "description": "A named build target, as specified in the \"configurations\" section of angular.json.\nEach named target is accompanied by a configuration of option defaults for that target.",
           "type": "string",
           "aliases": [
             "c"
           ]
         },
         "prod": {
-          "description": "When true, sets the build configuration to the production environment.\nAll builds make use of bundling and limited tree-shaking, A production build also runs limited dead code elimination using UglifyJS.",
+          "description": "When true, sets the build configuration to the production target.\nAll builds make use of bundling and limited tree-shaking, A production build also runs limited dead code elimination using UglifyJS.",
```
Comment (hansl) [inline packages/angular/cli/commands/definitions.json]: `production configuration`. The target is chosen by the command (`ng build` uses the `build` target), etc.

File: packages/angular/cli/commands/run.json (hunk 1 above)
Comment (hansl) [reply packages/angular/cli/commands/run.json]: I would use `Builder` when it's not a build command, yes. So in this case yes, that's a good idea.

Comment (jbogarthyde) [reply packages/angular/cli/commands/run.json]: Is `Builder` another tool or technical term that we need to introduce?

**Comment by jbogarthyde** [comment]
Please provide an example using the `run` command with the `universal` target.

**Comment by jbogarthyde** [comment]
@hansl I have addressed your requested changes - please check and remove blocker.
@alan-agius4 
Do either of you know why it's failing continuous-integration?

File: packages/angular/cli/commands/definitions.json (hunk 3 above)
Comment (alan-agius4) [inline packages/angular/cli/commands/definitions.json]: Should we change this to Terser instead of UglifyJS?

Comment (jbogarthyde) [reply packages/angular/cli/commands/definitions.json]: Removing tool reference.

File: packages/angular/cli/commands/run-long.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+For example, the "server" target for a newly generated app has a predefined
+alternate configuration named "production".
+
+You can define new targets and their configuration options in the "architect" section
+of the `angular.json` file.
+If you do so, you can run them from the command line using the `ng run` command.
+Execute the command using the following format.
+
+```
+ng run project:target[.configuration]
```
Comment (alan-agius4) [inline packages/angular/cli/commands/run-long.md]: Should be `ng run project:target[:configuration]`

**Comment by jbogarthyde** [comment]
Moved these commits to docs-preview branch - PR Docs preview #12837

**Pull request closed without merging**